README.md

```diff
@@ -1214,10 +1214,11 @@ backend {
 }
 ```
 
-If the backend supports docker, another optional configuration key `backend.providers.<backend>.config.submit-docker` may be specified. When the WDL contains a docker runtime attribute, this command will be provided the two additional variables:
+If the backend supports docker, another optional configuration key `backend.providers.<backend>.config.submit-docker` may be specified. When the WDL contains a docker runtime attribute, this command will be provided three additional variables:
 
 * `docker` - The docker image name.
 * `docker_cwd` - The path where `cwd` should be mounted within the docker container.
+* `docker_cid` - The host path to which the container ID file should be written.
```

Comment:
TOL should this go in the Release Notes? Maybe under a section "Important fixes", as this has been broken for a while.

Reply:
Sounds good to me.

Reply:
I stared at this for a a minute too.  A link to the reference doc would be helpful too: 
https://docs.docker.com/engine/reference/run/#pid-equivalent